apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/add-edit-notification-dialog.component.html

```diff
@@ -215,7 +220,7 @@ <h1 mat-dialog-title>{{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.TITLE_EDIT' | transl
           *ngIf="!data.createMailNotification"
           class="ml-2"
           color="accent"
-          [disabled]="loading || !editAuth"
+          [disabled]="loading || !editAuth || inputFormGroup.invalid"
```

Comment:
Please add a tooltip as with the other components (it's very useful when a user on the plaintext tab and the button is disabled because of something on the HTML tab)